pkgs/applications/editors/gnome-latex/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, vala, autoconf, autoconf-archive, automake, pkgconfig, intltool
+, gnome2, gnome3, appstream-glib, gobjectIntrospection, libtool, gtksourceview4
+, tepl, amtk, itstool, libxml2, glib, wrapGAppsHook }:
+let
+  version = "3.28.1";
+in stdenv.mkDerivation {
+  name = "gnome-latex-${version}";
+  src = fetchTarball {
+    url = "https://gitlab.gnome.org/GNOME/gnome-latex/-/archive/${version}/gnome-latex-${version}.tar.bz2";
+    sha256 = "17rbwg0v3m0myl797vrynk7wpz1fysp5px45bl7l4fpqwan2vgv8";
+  };
+
+  NIX_CFLAGS_COMPILE = "-I${glib.dev}/include/gio-unix-2.0";
+  nativeBuildInputs = [
+    autoconf
+    autoconf-archive
+    automake
+    libtool
+    wrapGAppsHook
+    pkgconfig
+    intltool
+  ];
+
+  buildInputs = [
+    vala
```

Comment:
The following should go to `nativeBuildInputs`: vala, appstream-glib, gobjectIntrospection, gnome2.gtkdoc, gnome3.yelp-tools, itstool